src/main/java/org/spongepowered/common/service/user/UserDiscoverer.java

```diff
@@ -196,6 +203,21 @@ private static User getFromStoredData(UUID uniqueId) {
         }
     }
 
+    private static Optional<org.spongepowered.api.profile.GameProfile> getProfileFromServer(UUID uuid) {
+        CompletableFuture<org.spongepowered.api.profile.GameProfile> gameProfile = Sponge.getServer().getGameProfileManager().get(uuid);
+
+        try {
+            org.spongepowered.api.profile.GameProfile profile = gameProfile.get();
+            if (profile != null) {
+                return Optional.of(profile);
+            } else {
+                return Optional.empty();
+            }
+        } catch (InterruptedException | ExecutionException e) {
```

Comment:
Why are you just swallowing the exception? This needs to be logged for any reason.